packages/valory/agents/apy_estimation/aea-config.yaml

```diff
@@ -95,12 +95,11 @@ models:
       history_start: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_HISTORY_START:int:1652544875}
       optimizer:
         n_trials: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_N_TRIALS:int:10}
-        timeout: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_TIMEOUT:int:60}
+        timeout: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_TIMEOUT:int:300}
         n_jobs: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_N_JOBS:int:1}
-        show_progress_bar: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_SHOW_PROGRESS_BAR:bool:False}
+        show_progress_bar: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_SHOW_PROGRESS_BAR:bool:false}
         scoring: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_SCORING:str:pinball}
         alpha: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_OPTIMIZER_ALPHA:float:0.25}
-      pair_ids: ${SKILL_APY_ESTIMATION_ABCI_MODELS_PARAMS_ARGS_PAIR_IDS:dict:null}
```

Comment:
Removed because I was getting:

```
Error: Attribute `models.params.args.pair_ids` is not allowed to be updated!
```

Reply:
In order to be able to update, you need to have it defined on the skill too.

Reply:
It is already defined on the skill, however, dict overrides are not allowed in the above way. Also, we cannot override the keys, which could be an issue, since someone might want to define a new DEX via an override. Any idea on how I could update the config's format to achieve this @angrybayblade?

https://github.com/valory-xyz/open-autonomy/blob/68257c07c87128b0589823d5b037113c2279da8d/packages/valory/skills/apy_estimation_abci/skill.yaml#L124-L129

Reply:
Ok - maybe that's because the default is `null` and not `dict`? Not sure what the constraints were there again

Reply:
@angrybayblade

Reply:
@Adamantios I think you can define them as a 2D list like `[["key", "value"]]`

Reply:
> @Adamantios I think you can define them as a 2D list like `[["key", "value"]]`

[This is not working](https://github.com/valory-xyz/open-autonomy/pull/1189) as it breaks tests. As it is a list, we get: `AttributeError: 'list' object has no attribute 'keys'` on behaviour unit tests.

Reply:
Fixed in #1189.